velox/vector/BaseVector.cpp

```diff
@@ -335,16 +335,12 @@ VectorPtr BaseVector::createInternal(
 }
 
 void BaseVector::addNulls(const uint64_t* bits, const SelectivityVector& rows) {
```

Comment:
Can we get rid of this, is there a case where we pass null buffer that is not part of BaseVector ?

Reply:
yes the current usages are a mix of buffers from nullBuffers in base vector and from selectivity vectors